### PR TITLE
Add dumps methods

### PR DESCRIPTION
**Pull request by shokme**
add support of [dumps](https://docs.meilisearch.com/references/dump.html#create-a-dump)

### CODE REVIEW

File: .code-samples.meilisearch.yaml [hunk 1]
```diff
@@ -315,3 +315,7 @@ faceted_search_walkthrough_facet_filters_1: |-
   $client->getIndex('movies')->search('thriller', ['facetFilters' => [['genres:Horror', 'genres:Mystery']], 'director' => "Jordan Peele"]);
 faceted_search_walkthrough_facets_distribution_1: |-
   $client->getIndex('movies')->search('Batman', ['facetsDistribution' => ['genres']);
+post_dump_1: |-
+  $client->createDump();
+get_dump_status_1: |-
+  $client->getDumpStatus('dump_uid');
```
Comment (curquiza) [inline .code-samples.meilisearch.yaml:321]:
```suggestion
  $client->getDumpStatus('20201101-110357260');
```

**Review by curquiza — CHANGES_REQUESTED** [review]
Awesome! Thank you so much @shokme!
I request a tiny change in the code-samples according to this PR in the docs I've just made https://github.com/meilisearch/documentation/pull/548 😇

**Comment by shokme** [comment]
Funny error :smile: 
I will try to do it another way, I wasn't aware of this potential error

**Comment by curquiza** [comment]
Better to test both methods as you did since it writes on the file system of the machine and can lead to random issues 👍
Merging!

**Review by curquiza — APPROVED** [review]

**Pull request merged**